docs/content/docs/custom-resource/autoscaler.md

```diff
@@ -260,17 +260,46 @@ job.autoscaler.metrics.window : 3m
-following command.
+following command. Please download released autoscaler-standalone jar from 
+[here](https://repo.maven.apache.org/maven2/org/apache/flink/flink-autoscaler-standalone/) first.
 
 ```
 java -cp flink-autoscaler-standalone-{{< version >}}.jar \
 org.apache.flink.autoscaler.standalone.StandaloneAutoscalerEntrypoint \
---flinkClusterHost localhost \
---flinkClusterPort 8081
+--autoscaler.standalone.fetcher.flink-cluster.host localhost \
+--autoscaler.standalone.fetcher.flink-cluster.port 8081
 ```
 
-Updating the `flinkClusterHost` and `flinkClusterPort` based on your flink cluster.
-In general, the host and port are the same as Flink WebUI.
+Updating the `autoscaler.standalone.fetcher.flink-cluster.host` and `autoscaler.standalone.fetcher.flink-cluster.port` 
+based on your flink cluster. In general, the host and port are the same as Flink WebUI.
+
+### Using the JDBC Autoscaler State Store
+
+A driver dependency is required to connect to a specified database. Here are drivers currently supported, 
+please download JDBC driver and initialize database and table first.
+
+| Driver     | Group Id                   | Artifact Id            | JAR                                                                             | Schema                  |
+|:-----------|:---------------------------|:-----------------------|:--------------------------------------------------------------------------------|-------------------------|
+| MySQL      | `mysql`                    | `mysql-connector-java` | [Download](https://repo.maven.apache.org/maven2/mysql/mysql-connector-java/)    | [Table DDL](https://github.com/apache/flink-kubernetes-operator/blob/main/flink-autoscaler-plugin-jdbc/src/main/resources/schema/mysql_schema.sql)     |
+| PostgreSQL | `org.postgresql`           | `postgresql`           | [Download](https://jdbc.postgresql.org/download/)                               | [Table DDL](https://github.com/apache/flink-kubernetes-operator/blob/main/flink-autoscaler-plugin-jdbc/src/main/resources/schema/postgres_schema.sql)  |
+| Derby      | `org.apache.derby`         | `derby`                | [Download](http://db.apache.org/derby/derby_downloads.html)                     | [Table DDL](https://github.com/apache/flink-kubernetes-operator/blob/main/flink-autoscaler-plugin-jdbc/src/main/resources/schema/derby_schema.sql)     |
+
+```
+JDBC_DRIVER_JAR=./mysql-connector-java-8.0.30.jar
+# export the password of jdbc state store
+export STATE_STORE_JDBC_PWD=123456
+
+java -cp flink-autoscaler-standalone-{{< version >}}.jar:${JDBC_DRIVER_JAR} \
+org.apache.flink.autoscaler.standalone.StandaloneAutoscalerEntrypoint \
+--autoscaler.standalone.fetcher.flink-cluster.host localhost \
+--autoscaler.standalone.fetcher.flink-cluster.port 8081 \
+--autoscaler.standalone.state-store.type jdbc \
+--autoscaler.standalone.state-store.jdbc.url jdbc:mysql://localhost:3306/flink_autoscaler \
+--autoscaler.standalone.state-store.jdbc.username root
```

Comment:
should we also state the password-env-variable config here?

Reply:
In general, the environment variable name doesn't need to be changed. Users need to export the password using this environment variable.

So I didn't mention `password-env-variable` here. But in the beginning of this doc, it mentioned how to export password. WDYT?

```
# export the password of jdbc state store
export STATE_STORE_JDBC_PWD=123456
```

Reply:
You are right, sorry I got confused by the logic for second :) 

I think it would make sense to keep the config for the password there as well as an option for people to use instead of the env. The env variable is great for prod but for testing the password may be more convenient. Wdyt?

Reply:
Do you mean we provide the `autoscaler.standalone.state-store.jdbc.password-env-variable` and `autoscaler.standalone.state-store.jdbc.password` together?

- For prod, we recommend users to `export STATE_STORE_JDBC_PWD=xxx`.
- For testing, using `autoscaler.standalone.state-store.jdbc.password` is convenient.

Right?

I'm afraid 2 options let users get confused here. 

Also, if we provide 2 options, we need to consider the priority. Which one will be used if users set 2 options together.

Reply:
About testing, if we only test `JdbcAutoScalerStateStore`, the password is easy to be set.

JdbcAutoScalerStateStore will work well regardless of whether the Autoscaler standalone uses  or the Kubernetes-operator uses if `JdbcAutoScalerStateStore` is tested properly. 

So test `JdbcAutoScalerStateStore` in Autoscaler standalone isn't necessary, right?

If so, we don't need the `autoscaler.standalone.state-store.jdbc.password` for now.  And we can add it if we really needs it in the future.

WDYT?

Reply:
okay, sounds good